lib/resty/acme/autossl.lua

```diff
@@ -359,6 +367,16 @@ function AUTOSSL.init(autossl_config, acme_config)
   acme_config.storage_adapter = autossl_config.storage_adapter
   acme_config.storage_config = autossl_config.storage_config
 
+  if acme_config.storage_adapter == "resty.acme.storage.redis" and
+    acme_config.storage_config.namespace then
+    local namespace = acme_config.storage_config.namespace
+    for _, v in ipairs(reserved_words) do
+      if namespace:find(v, 1, true) == 1 then
```

Comment:
we can do a `namespace:sub(1, #v) == v`, it looks clearer. alternatively we can also just `local re = "^(" .. tabel.concat(reserved_words, '|') .. ")"` to make a regex to match it.
both will make the code simpler